app/src/main/java/com/sopt/clody/presentation/ui/diarylist/screen/DiaryListScreen.kt

```diff
@@ -44,14 +47,25 @@ fun DiaryListRoute(
         diaryListViewModel.fetchMonthlyDiary(selectedYearInDiaryList, selectedMonthInDiaryList)
     }
 
+    when (diaryDeleteState) {
+        is DiaryDeleteState.Success -> {
+            LaunchedEffect(Unit) {
+                diaryListViewModel.fetchMonthlyDiary(selectedYearInDiaryList, selectedMonthInDiaryList)
+            }
+        }
+
+        else -> {}
+    }
+
```

Comment:
```suggestion
LaunchedEffect(diaryDeleteState) {
    if (diaryDeleteState is DiaryDeleteState.Success) {
        diaryListViewModel.fetchMonthlyDiary(selectedYearInDiaryList, selectedMonthInDiaryList)
    }
}
```
[**P3**]
이렇게 하는 것도 좋아보이는듯!